homeassistant/components/mqtt/device_tracker/schema_yaml.py

```diff
@@ -53,5 +76,10 @@ def async_message_received(msg, dev_id=dev_id):
             hass.async_create_task(async_see(**see_args))
 
         await async_subscribe(hass, topic, async_message_received, qos)
+        subscriptions.append(
+            await async_subscribe(hass, topic, async_message_received, qos)
+        )
+
+    config_entry.async_on_unload(_entry_unload)
```

Comment:
What happens after a reload, are the legacy device trackers gone or are they setup again?